docs/source/endog_exog.rst

```diff
@@ -79,7 +79,7 @@ It is up to the user to know (or to consult a text book to find out) what the
 underlying statistical assumptions for the models are. As an example, ``exog``
 in ``OLS`` can have lagged dependent variables if the error or noise term is
 independently distributed over time (or uncorrelated over time). However, if
-the error terms are autocorrelated, then OLS does not have good statistical
+the error terms are autocorrelated in the presense of lagged dependent variables, then OLS does not have good statistical
```

Comment:
Could you please wrap this using a line length similar to the others.  Thanks for the clarification.